enhancements/insights/insights-operator-runtime.md

```diff
@@ -0,0 +1,542 @@
+
+In many cases, the image ID and its layers do not provide any meaningful information about the actual application or application language/platform/framework...  
+To get this level of information, we can gather additional data on the running containers.   
+By inspecting running containers, the Insights Operator can enrich the gathered data and capture the kind of workload that is running on OpenShift. This information can be used by Red Hat to target  OpenShift investments at the most popular workloads, inform decisions around long-life offerings for application platforms and frameworks, identify growth opportunities in emerging workloads, and measure the success of our strategic initiatives in AI and Middleware.
+
+For example, inspecting the container allows to gather additional data such as:
+
+* The container (with the ID `sha256:5edd4b...`) is based on __RHEL 8.9__ ands runs a __Kafka Broker 3.6.1__ with __Java OpenJDK 17.0.9__
+* The container (with the ID `sha256:4redc...`)  is based on __Fedora 17__, and runs a __Quarkus 3.6.2__ application with __Java Eclipse Adoptium 21.0.1__ 
+* The container (with the ID `sha256:7euc...`)  is based on __RHEL 8.8__, and runs a __Red Hat Build of Quarkus 3.2.13__ application with __Red Hat Java 17.0.3-1__ 
+* The container (with the ID `sha256:2badc...`)  is based on __Alpine Linux v3.11__, and runs a __Spring Boot 2.7__ with __Java OpenJDK 17.0.8.1__
+* The container (with the ID `sha256:3fdb45...`) is based on __Ubuntu 20.04.4 LTS__, and runs a __Node.js v18.18.2__ application
+* The container (with the ID `sha256:5edd4b...`) is a scratch container (not layered on top of any image) and runs a __Go 1.18.10__ executable
+* The container (with the ID `sha256:566tr...`) is  based on __RHEL 8.9__ ands runs __PostgreSQL 10.23__
+
+This enhancement proposes to extract this additional layer of data to get deeper insights on the OpenShift workload.
+It proposes to do so while keeping the existing constraints and requirements that the Insights Operator already has for privacy, security & performance.
+
+## Motivation
+
+Currently, the Insights operator tries to identify what workload is used on OpenShift by looking for known fingerprinting in layers of containers, but there are a lot of issues with this approach where we can only identify base-level containers.
+We do, for example, not know if a python container is using AI libraries.
+Another issue is that we can only identify containers using known base containers (like OpenJDK), but we do not know what type of workload is more popular (Spring Boot, Quarkus, WildFly, EAP, etc.).
+
+The Insights operator can also not tell if a container is using a non-Red Hat provided base containers like Ubuntu or Debian. Knowing this data means that BU and product leadership can use this data to prioritise future work and realign investment.
```

Comment:
`prioritise` -> `prioritize`

Reply:
I can't help but writing UK English instead of US English :)